schemas/view-schema.json

```diff
@@ -181,6 +186,20 @@
               },
               "readOnly": true
             },
+            "frozen_frame": {
+              "type": "object",
```

Comment:
do we plan to extend `frozen_frame` in future to have not only `count` field?

Reply:
No, not necessarily. We're following the standard set by the `action`, `error`, `crash`, `resource` and `long_task` fields

Reply:
FMU, frozen frames are long tasks, would it make sense to instead have 

`view.long_task.frozen_frames_count`

Reply:
We could do that as well, but for crashes (which are a subtype of errors) we do have `view.error.count` and `view.crash.count`, and not `view.error.crash_count`. IMO, it is better to have a `view.frozen_frame.count` as it's consistent with existing attribtues.

Reply:
ok, sounds good to me

Reply:
consistency is good, but it seems that using object with only `xxx.count` is not so useful, do we really need to keep adding new object each time ?

Reply:
I won't fight for it , but consistency makes it easier for our customer to find attributes. If we start changing the naming paradigm now, it might add confusions to our customers.

Reply:
ok, let's keep it :)